charts/tidb-backup/values.yaml

```diff
@@ -42,7 +42,7 @@ storage:
   size: 100Gi
 
 # backupOptions is the options of mydumper https://github.com/maxbube/mydumper/blob/master/docs/mydumper_usage.rst#options
-backupOptions: "--verbose=3"
+backupOptions: "-t 16 -F 256 --skip-tz-utc --verbose=3"
```

Comment:
Add simple descriptions for these default options.
- `-t` is thread count, larger thread count will speed up the backup, but may impact the performance of the upstream TiDB.
- `-F` is the chunk size, a big table is partitioned into many chunks.

Other useful options are `-B` for database, and `-T` for tables. See https://github.com/maxbube/mydumper/blob/master/docs/mydumper_usage.rst#options for more options.

Reply:
done